entropix/torch_main.py

```diff
@@ -16,51 +18,71 @@
 from entropix.torch_weights import XfmrWeights, LayerWeights, load_weights
 from entropix.torch_sampler import sample
 from entropix.prompts import prompt, bp1
-
-
-def apply_scaling(freqs: torch.Tensor):
-    # Values obtained from grid search
-    scale_factor = 8
-    low_freq_factor = 1
-    high_freq_factor = 4
-    old_context_len = 8192  # original llama3 length
-
-    low_freq_wavelen = old_context_len / low_freq_factor
-    high_freq_wavelen = old_context_len / high_freq_factor
-    new_freqs = []
-    for freq in freqs:
-        wavelen = 2 * math.pi / freq
-        if wavelen < high_freq_wavelen:
-            new_freqs.append(freq)
-        elif wavelen > low_freq_wavelen:
-            new_freqs.append(freq / scale_factor)
-        else:
-            assert low_freq_wavelen != high_freq_wavelen
-            smooth = (old_context_len / wavelen - low_freq_factor) / (
-                high_freq_factor - low_freq_factor
+from entropix.main import precompute_freqs_cis as jax_precompute_freqs_cis
```

Comment:
think this line can be removed since precompute_freqs_cis is defined below

Reply:
yes, true. i have experimented with other approaches and this is a left over code from that

Reply:
added the commit to this pr below, also removed he jax imports, same concepts: have experimented with other approaches but ended up leaving the torch implementation intact